### ISSUE TITLE
运行报错 Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by yhwang-hub** [drilistbox/TRT_FlashOcc#2]
使用gdb调试报错如下所示：
`
root@c5970ab673f3:/home/uisee/FlashOCC/TRT_FlashOcc# gdb ./build/bevdemo                 
GNU gdb (Ubuntu 8.1.1-0ubuntu1) 8.1.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from ./build/bevdemo...done.
(gdb) set args ./flashocc.yaml 
(gdb) r
Starting program: /home/uisee/FlashOCC/TRT_FlashOcc/build/bevdemo ./flashocc.yaml 
warning: Error disabling address space randomization: Operation not permitted
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7f966ce5a700 (LWP 18463)]

GPU has cuda devices: 1
----device id: 0 info----
  GPU : NVIDIA GeForce RTX 3060 
  Capbility: 8.6
  Global memory: 12049MB
  Const memory: 64KB
  Shared memory in a block: 48KB
  warp size: 32
  threads in a block: 1024
  block dim: (1024,1024,64)
  grid dim: (2147483647,65535,65535)

Successful load config : ./flashocc.yaml!

[New Thread 0x7f9667fff700 (LWP 18464)]
[New Thread 0x7f96677fe700 (LWP 18465)]

Thread 1 "bevdemo" received signal SIGSEGV, Segmentation fault.
0x0000000000000000 in ?? ()
(gdb) bt
#0  0x0000000000000000 in ?? ()
#1  0x00007f969327c0c5 in nvjpegCreateEx () from /usr/local/cuda-11.1/lib64/libnvjpeg.so.11
#2  0x000055c602061492 in nvjpegDecoder::init() ()
#3  0x000055c602061bae in nvjpegDecoder::nvjpegDecoder(unsigned long, unsigned long) ()
#4  0x000055c6020555a8 in DataLoader::DataLoader(int, int, int, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, bool) ()
#5  0x000055c602034072 in TestNuscenes(YAML::Node&) ()
#6  0x000055c60203005e in main ()
`

**Comment by drilistbox** [comment]
Maybe the data is not prepared well?

**Comment by yhwang-hub** [comment]
> Maybe the dataset is not prepared well?

数据集准备好了，但是仍会报这个错

**Comment by drilistbox** [comment]
这个问题我也没遇到过，我也是cuda-11.1，你的tensorrt版本是啥

**Comment by yhwang-hub** [comment]
> 这个问题我也没遇到过，我也是cuda-11.1，你的tensorrt版本是啥

tensorrt版本8.5.1.7

**Comment by drilistbox** [comment]
加我微信吧，drilistbox，看看啥原因

**Comment by yhwang-hub** [comment]
问题已解决，是我自己docker环境的问题
